app/controllers/works_controller.rb

```diff
@@ -2,6 +2,7 @@ class WorksController < ApplicationController
   # We should always be able to tell what category
   # of work we're dealing with
   before_action :category_from_work, except: [:root, :index, :new, :create]
+  before_action :require_login, only: %i[index show create new destroy update]
```

Comment:
It's great that you added the filter to require a login, but you should update the tests to reflect this.